src/jsonrpc.rs

```diff
@@ -0,0 +1,25 @@
+use jsonrpc_stdio_server::{ServerBuilder, jsonrpc_core::*};
+use tracing::{info, trace};
+
+pub async fn start_server() {
+    info!("Starting jsonrpc server");
+    let server = new_server();
+    let server = server.build();
+    server.await;
+}
+
+pub fn new_server() -> ServerBuilder {
+    let mut io = IoHandler::default();
+    handle_ping(&mut io);
+    let request = r#"{"jsonrpc": "2.0", "method": "ping", "params": {}, "id": 1}"#;
+	let response = r#"{"jsonrpc":"2.0","result":"pong","id":1}"#;
+    assert_eq!(io.handle_request_sync(request), Some(response.to_string()));
```

Comment:
Remove it :cry:

Reply:
I suppose it should be placed in some test file. I'll move it elsewhere.

Reply:
Sad is that the posix_test seems a bit unstable, it keeps failing thus blocks me from putting `cargo test` to a pre-commit hook 😢